networks/motion/CNN3D/3D_CNN.py

```diff
@@ -0,0 +1,218 @@
+import os.path
```

Comment:
deprecated file
Please pull first current master from branch and merge it local with your code, then push online and request a pull to master

Reply:
same here